arkane/encorr/data.py

```diff
@@ -67,7 +67,9 @@
 # Iodine SOC calculated as a weighted average of the electronic spin splittings of the lowest energy state.
 # The splittings are obtained from Huber, K.P.; Herzberg, G., Molecular Spectra and Molecular Structure. IV.
 # Constants of Diatomic Molecules, Van Nostrand Reinhold Co., 1979
-SOC = {'H': 0.0, 'N': 0.0, 'O': -0.000355, 'C': -0.000135, 'S': -0.000893, 'P': 0.0, 'I': -0.011547226}
+# Spin orbit correction for F, Si, Cl, Br and B taken form https://cccbdb.nist.gov/elecspin.asp
+SOC = {'H': 0.0, 'N': 0.0, 'O': -0.000355, 'C': -0.000135, 'S': -0.000893, 'P': 0.0, 'I': -0.011547226,
+        'F': -0.000614, 'Si': -0.000682 , 'Cl': -0.001338, 'Br': -0.005597 , 'B': -0.000046 } 
```

Comment:
please correct white spaces: there's an etra one before `'F'`, and an extra one after `'B': -0.000046`.

Reply:
Fixed